api/hostmanagement/openamt/authorization.go

```diff
@@ -14,6 +14,14 @@ type authenticationResponse struct {
 	Token string `json:"token"`
 }
 
+func (service *Service) Authorization(configuration portainer.OpenAMTConfiguration) (string, error) {
+	token, err := service.executeAuthenticationRequest(configuration)
+	if err != nil {
+		return "", err
+	}
+	return token.Token, nil
+}
+
```

Comment:
Why not expose ExecuteAuthenticationRequest as Authorization?
They are basically the same thing.

And if you really need this function why not just 

```
func (service *Service) Authorization(configuration portainer.OpenAMTConfiguration) (string, error) {
   return service.executeAuthenticatonRequest(configuration)
}
```

Reply:
Agree!